README.md

```diff
@@ -1,56 +1,31 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# hello_microverse_project
 
-> Description the project.
+> Introduction to linters and Github flow
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Html
+- css
 
-## Live Demo (if available)
+## Live Demo 
 
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+[Live Demo Link](https://assadounto.github.io/MicroverseProject1/)
 
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
 
```

Comment:
- [ ] A professional README should include instruction steps to run the app locally for the sake of the non-technical user. Kindly include steps to clone and run the app locally.